harmony-fe/src/components/post/post.tsx

```diff
@@ -18,24 +29,101 @@ interface PostProps {
 }
 
 const Post = ({ title, name, numComments, numLikes }: PostProps) => {
+  const [isLiked, setIsLiked] = useState(false)
+  const [isCommentSectionOpen, setIsCommentSectionOpen] = useState(false)
+
+  const toggleLike = () => {
+    setIsLiked(prevState => !prevState)
+  }
+
+  const toggleCommentSection = () => {
+    setIsCommentSectionOpen(prevState => !prevState)
+  }
+
+  const [commentInput, setCommentInput] = useState('')
+  const [comments, setComments] = useState<string[]>([])
+
+  const handleCommentInputChange = (
+    event: React.ChangeEvent<HTMLInputElement>
+  ) => {
+    setCommentInput(event.target.value)
+  }
+
+  const handleCommentSubmission = () => {
+    if (commentInput.trim() !== '') {
+      setComments(prevComments => [...prevComments, commentInput])
+      setCommentInput('')
+      toggleCommentSection()
+    }
+  }
   return (
-    <Card variant="outlined" sx={{ minWidth: 275 }}>
+    <Card variant="outlined" sx={{ mb: 1 }}>
       <Stack direction="row">
         <CardContent>
           <Avatar sx={{ bgcolor: deepOrange[500] }}>poll</Avatar>
         </CardContent>
-        <Box>
+
+        <Box sx={{ width: '100%' }}>
           <CardContent>
-            <TextBlock gutterBottom variant="h5">
-              {title}
-            </TextBlock>
+            <Link href="/posts/${[postId]}">
```

Comment:
I think this should like this:
```suggestion
            <Link href={`/posts/${postId}`}>
```
You need to use \` as the quotes to use the ${} stuff, right now if postId=100 this link will go to `/posts/${[postId]}` instead of `/posts/100`

Reply:
I noticed that but couldn't figure out how to resolve that. I'll change that. Thanks!